config/default.yml

```diff
@@ -475,6 +475,12 @@ Capybara/FeatureMethods:
   EnabledMethods: []
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Capybara/FeatureMethods
 
+Capybara/VisibilityMatcher:
+  Description: Checks for boolean visibility in capybara finders.
+  Enabled: true
+  VersionAdded: '0.81'
```

Comment:
No other cop seems to have it, should this be removed or use the rubocop-rspec version rather than the rubocop version number?

Reply:
We have a ticket and a pull request for adding this for all the cops https://github.com/rubocop-hq/rubocop-rspec/issues/699 https://github.com/rubocop-hq/rubocop-rspec/pull/721
It's safe to say it's `1.39` here, next `rubocop-rspec` minor release.